src/test/style.ts

```diff
@@ -89,44 +88,46 @@ describe('style', function() {
     assert.equal(elm.style.getPropertyValue('--myVar'), 2);
   });
   it('implicially removes css variables from element', function() {
-    var vnode1 = h('div', [h('i', {style: {'--myVar': 1}})]);
+    var vnode1 = h('div', [h('i', {style: {'--myVar': 1 as any}})]);
     var vnode2 = h('div', [h('i')]);
-    var vnode3 = h('div', [h('i', {style: {'--myVar': 2}})]);
+    var vnode3 = h('div', [h('i', {style: {'--myVar': 2 as any}})]);
     patch(vnode0, vnode1);
     assert.equal(elm.firstChild.style.getPropertyValue('--myVar'), 1);
     patch(vnode1, vnode2);
     assert.equal(elm.firstChild.style.getPropertyValue('--myVar'), '');
     patch(vnode2, vnode3);
     assert.equal(elm.firstChild.style.getPropertyValue('--myVar'), 2);
   });
-  it('updates delayed styles in next frame', function() {
-    var patch = snabbdom.init([
-      require('../modules/style').default,
-    ]);
-    var vnode1 = h('i', {style: {fontSize: '14px', delayed: {fontSize: '16px'}}});
-    var vnode2 = h('i', {style: {fontSize: '18px', delayed: {fontSize: '20px'}}});
+  it('updates delayed styles in next frame', function(done) {
+    var vnode1 = h('i', {style: {fontSize: '14px', delayed: {fontSize: '16px'} as any}});
+    var vnode2 = h('i', {style: {fontSize: '18px', delayed: {fontSize: '20px'} as any}});
     elm = patch(vnode0, vnode1).elm;
     assert.equal(elm.style.fontSize, '14px');
-    fakeRaf.step();
-    fakeRaf.step();
-    assert.equal(elm.style.fontSize, '16px');
-    elm = patch(vnode1, vnode2).elm;
-    assert.equal(elm.style.fontSize, '18px');
-    fakeRaf.step();
-    fakeRaf.step();
-    assert.equal(elm.style.fontSize, '20px');
+    requestAnimationFrame(() => {
+      requestAnimationFrame(() => {
+        assert.equal(elm.style.fontSize, '16px');
+        elm = patch(vnode1, vnode2).elm;
+        assert.equal(elm.style.fontSize, '18px');
+        requestAnimationFrame(() => {
+          requestAnimationFrame(() => {
+            assert.equal(elm.style.fontSize, '20px');
+            done()
+          })
+        })
+      })
+    })
```

Comment:
I little bit more indented, but works. One thing I don't understand is why it takes two calls to read the change. Anyone?